src/isAnnotatedForRemoval.js

```diff
@@ -0,0 +1,7 @@
+// @flow weak
+
+export default function (node) {
+  const comments = node.trailingComments || [];
```

Comment:
A leading comment would be an [alternative](http://astexplorer.net/#/gist/85f95a9dcb9f7ebf5577b163fcec99c2/415c7c4718e4d18479c2e905d912e1bd71771fa0):
```js
Foo1.propTypes = { // remove-proptypes
  bar1: PropTypes.string.isRequired,
}


class Foo2 extends React.Component {
  static propTypes = { // remove-proptypes
    bar2: PropTypes.string,
  };

  render() {}
}

```